src/test/scala/uk/gov/hmrc/ct/utils/Mocks.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.utils
+
+import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
+import org.scalatestplus.mockito.MockitoSugar
+
+trait Mocks extends MockitoSugar {
+  val mockComputationsBoxRetriever: ComputationsBoxRetriever = mock[ComputationsBoxRetriever]
```

Comment:
You could argue this is futile, but I envisioned something similar to what we have in Cato Frontend